git-remote-mediawiki.perl

```diff
@@ -724,6 +729,10 @@ sub fe_escape_path {
     $path =~ s/\\/\\\\/g;
     $path =~ s/"/\\"/g;
     $path =~ s/\n/\\n/g;
+    if ($use_subpage_dirs) {
+        $path =~ s/%2F((?:%2F)*)/"\/" . $1/ge;
+        $path =~ s/^(\/+)/"%2F" x length $1/ge;
```

Comment:
those regex would be more readable if you used another seperator than `/`. i often use `#` for that purpose.

Reply:
how are we protecting against directory transversal attacks here?

Reply:
@anarcat Regex: Sorry, I was not aware of that.

_how are we protecting against directory transversal attacks here?_

Not yet.

Reply:
okay, let's make sure we protect against directory transversal before we merge.